# Sign's position and direction 
float64 x
float64 y
float64 th
---
bool isDone
